mkdocs.yml

```diff
@@ -71,3 +72,8 @@ extra:
   analytics:
     provider: google
     property: !ENV [GOOGLE_ANALYTICS]
+
+markdown_extensions:
```

Comment:
this should be with all the other extensions, also a quick comment / link to why we want it.